migrator.sh

```diff
@@ -556,13 +556,13 @@ query_source_images() {
       IMAGE_TAGS=""
 
       # set page URL to start with
-      PAGE_URL="https://hub.docker.com/v2/repositories/${NAMESPACE}/${i}/tags/?page=1&page_size=250"
+      PAGE_URL="https://hub.docker.com/v2/repositories/${NAMESPACE}/${i}/tags/?page=1&page_size=${NUM_TAGS}"
```

Comment:
What's the purpose of changing the number of tags to return here?  This is just to ensure that we are not querying the API too much/too little.